### PR TITLE
Hotfix/transparency

### PR DESCRIPTION
**Pull request by xtrm-en**
## Description
:pleading_face: 

## Checklist
- [x] I made a clear description of what was changed
- [x] I stated why these changes were necessary
- [x] I updated documentation or said what needs to be updated
- [x] I made sure these changes are backwards compatible
- [x] This pull request is for one feature/bug fix

### CODE REVIEW

File: src/main/java/cc/polyfrost/oneconfig/config/annotations/Slider.java [hunk 1]
```diff
@@ -44,7 +44,7 @@
 
     float max();
 
-    int step() default 0;
+    float step() default 0;
```
Comment (Wyvest) [inline src/main/java/cc/polyfrost/oneconfig/config/annotations/Slider.java:47]: change this in Number too

**Review by Wyvest — CHANGES_REQUESTED** [review]

**Comment by romangraef** [comment]
image
Bruh you did none of these things

**Comment by xtrm-en** [comment]
> image Bruh you did none of these things

skill issue

**Review by Wyvest — APPROVED** [review]

**Review by Rattlyy — APPROVED** [review]
yes

**Pull request merged**